packages/manager/modules/nutanix/src/onboarding/template.html

```diff
@@ -19,7 +19,14 @@ <h4 data-translate="nutanix_onboarding_short_description"></h4>
             target="_blank"
             rel="noopener"
         >
-            <span data-translate="nutanix_onboarding_order"></span>
+            <span
+                ng-if="$ctrl.ovhSubsidiary === 'US'"
+                data-translate="nutanix_onboarding_contact_us"
+            ></span>
+            <span
+                ng-if="$ctrl.ovhSubsidiary !== 'US'"
+                data-translate="nutanix_onboarding_order"
+            ></span>
```

Comment:
shouldn't this be updated to use ctaLabel ?

Reply:
If we want do that we need to do translate in `index.js`and not `routing.js`. We think is not the easier to maintain if we put the translation login on it